Makefile

```diff
@@ -0,0 +1,11 @@
+start: ## Start the devstack
+	docker-compose -f docker-compose.yml up -d
```

Comment:
@ChrisChV `-f docker-compose.yml` can be omitted, because this is the default value.